popup.js

```diff
@@ -0,0 +1,204 @@
+            </div>
+            <div class="popup-image4"></div>
+            <div>
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Praesentium et deserunt fuga nulla dolorem iure vel, itaque corrupti molestiae iusto?</p>
+            </div>
+            <div>
+                <a href="#">See live</a>
+                <a href="#">See source</a>
+                <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsum llat iias tempora illo suscipit optio, eligendi esse ullam fuga non facere dolorem eveniet mlaudantium omnis maiores eos corporis eum temporibus est, vel fuga modi ullam. Qui eaque dolores nesciunt aliquid cum! </p>
+            </div>
+        </div> 
+    <!-- proyecct5 -->
+    <div class="popup popup5">
+        <div class="close-container">
+        <div class="close-x"></div>
+    </div>
+            <h2 class="popup-title">Project name goes here</h2>
+            <div class="popup-skils">
+                <h3  class="popup-h3">HTML/CSS</h3>
+                <h3 class="popup-h3">Ruby on Rails</h3>
+                <h3 class="popup-h3">JavaScript</h3>        
+            </div>
+            <div class="popup-image5"></div>
+            <div>
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Praesentium et deserunt fuga nulla dolorem iure vel, itaque corrupti molestiae iusto?</p>
+            </div>
+            <div>
+                <a href="#">See live</a>
+                <a href="#">See source</a>
+                <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsum llat iias tempora illo suscipit optio, eligendi esse ullam fuga non facere dolorem eveniet mlaudantium omnis maiores eos corporis eum temporibus est, vel fuga modi ullam. Qui eaque dolores nesciunt aliquid cum! </p>
+            </div>
+        </div> 
+    <!-- proyecct6 -->
+    <div class="popup popup6">
+        <div class="close-container">
+        <div class="close-x"></div>
+    </div>
+            <h2 class="popup-title">Project name goes here</h2>
+            <div class="popup-skils">
+                <h3  class="popup-h3">HTML/CSS</h3>
+                <h3 class="popup-h3">Ruby on Rails</h3>
+                <h3 class="popup-h3">JavaScript</h3>        
+            </div>
+            <div class="popup-image6"></div>
+            <div>
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Praesentium et deserunt fuga nulla dolorem iure vel, itaque corrupti molestiae iusto?</p>
+            </div>
+            <div>
+                <a href="#">See live</a>
+                <a href="#">See source</a>
+                <p> Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsum llat iias tempora illo suscipit optio, eligendi esse ullam fuga non facere dolorem eveniet mlaudantium omnis maiores eos corporis eum temporibus est, vel fuga modi ullam. Qui eaque dolores nesciunt aliquid cum! </p>
+            </div>
+        </div> 
```

Comment:
- [Optional] kindly works data has to be stored in JavaScript objects containing the information for each project. name, description, featured image, technologies, link to live version, and link to the project source code. Ie: 👌👌

```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
},{...
]
````
👌👌